pkg/jx/cmd/create_addon_cloudbees.go

```diff
@@ -75,6 +76,7 @@ func NewCmdCreateAddonCloudBees(commonOpts *CommonOptions) *cobra.Command {
 	}
 
 	cmd.Flags().BoolVarP(&options.Sso, "sso", "", false, "Enable single sign-on")
+	cmd.Flags().StringVarP(&options.DefaultRole, "default-role", "", "", "The default role to apply to new users. Defaults to no role and applies to the admin namespace only")
```

Comment:
(nit) - IMHO its nicer to put these flags into constants so that they rename more safely if anyone does so in the future.